mmv1/third_party/terraform/services/compute/universe_domain_template_test.go

```diff
@@ -0,0 +1,68 @@
+package compute_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccUniverseDomainDisk(t *testing.T) {
```

Comment:
Can we put these inside a dedicated package? That will mean that we can test that package on its own rather than need to record a list of the universe domain tests elsewhere.

Reply:
Sure